v2/CHANGES

```diff
@@ -16,7 +16,7 @@
             with a strongly-typed metadata struct or interface.
 - [x] Change Request and Response body to yarpc.Buffer type
       - [x] Accumulate request bodies at the inbound
-- [ ] yarpcrouter.Router needs to accept middleware to decorate registered
+- [x] yarpcrouter.Router needs to accept middleware to decorate registered
```

Comment:
Is this a bad time to mention that I’ve reconsidered this stance? I think we should construct all the middleware chains at the Fx layer and just provide the component combinators in the `yarpc` API layer.

Reply:
I believe that this is what we're doing, see my response below

Reply:
👓 Ah, I see.